Describe aws/gcp actions to configure IAM roles/Workload ID for the O… (#353)
okteto/docs: src/content/self-hosted/administration/configuration.mdx

### publicOverride

> Use this property to override the Public URL where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with your [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of choice.

Reviewer: ```suggestion
Use this property to override the Public URL where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) you provide.
```

##### IAM Role for Service Account (IRSA)

> Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply these changes to the Okteto configuration:

Reviewer: Can we link to official docs on this?

Reviewer: Also, can we specify the exact permissions that we require for the IAM?

> Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply these changes to the Okteto configuration:

Reviewer: Can we also tell the reader the exact name of the service account? Or show them how to get it?

> Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply these changes to the Okteto configuration:

Reviewer: ```suggestion
Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply the changes below to your Helm configuration file:
```

Reviewer: We also need to tell them to upgrade no?

> Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply the changes below to your Helm configuration file:

Reviewer: This reads like we are in a rush :P. We need to take our time and explain every step.  We don't have to write everything ourselves, we can send them to the corresponding docs on the cloud provider's doc. But we need to fully document the exact permissions needed for the role, people always get this wrong. 

The same goes for the GCP section.

      memory: 50Mi
```

#### buildCredentialHelpers

Configure IAM roles to pull private images from your Dockerfiles. Disabled if [privateRegistry](self-hosted/administration/configuration.mdx#privateregistry) is defined.

> For private ECRs, add a mapping of registry hostnames as keys to credential helpers as values. Registry hostnames must be Fully Qualified Domain Names.

Reviewer: What's an ECR?

Reviewer: What are the accepted values for the credential helpers?

```yaml
installer:
  buildCredentialHelpers:
    111122223333.dkr.ecr.eu-central-1.amazonaws.com: ecr-login
```

> Create an IAM role with access to your private ECR, assume this role by the audience of Okteto Developer Service Accounts (service accounts in the `okteto` namespace) and apply these changes to the Okteto configuration:

Reviewer: We need to explain how to do this. Is not trivial and it's not that standard yet.
